.ci_support/linux_64_.yaml

```diff
@@ -1,9 +1,9 @@
 cdt_name:
 - cos6
 channel_sources:
-- conda-forge,defaults
+- conda-forge
 channel_targets:
-- conda-forge main
+- conda-forge diffpriv_dev
```

Comment:
Is DiffPriv listing on a separate channel?
This is not advisable as it would create unnecessary environment issues; sticking to just `conda-forge` and `main` is best.

Reply:
No, the diffpriv_dev is for pre-releases of the diffpriv package. the main channel isn't where we want to publish the package.